### ISSUE TITLE
Drupal 8.5 and Drupal console 1.8.0 issues on running commands

### ISSUE THREAD

**Issue by cr0ss** [hechoendrupal/drupal-console#3891]
I'm experiencing  following issue:

../vendor/bin/drupal --version
console.dotenv_init_generator - Class Drupal\Console\Dotenv\Generator\InitGenerator does not exist
ArgumentCountError: Too few arguments to function Drupal\Console\Core\Utils\ChainDiscovery::__construct(), 2 passed and exactly 4 expected in /home/vagrant/builtinx/vendor/drupal/console-core/src/Utils/ChainDiscovery.php on line 83 #0 [internal function]: Drupal\Console\Core\Utils\ChainDiscovery->__construct('/home/vagrant/b...', Object(Drupal\Console\Core\Utils\ConfigurationManager))
#1 /home/vagrant/builtinx/vendor/symfony/dependency-injection/ContainerBuilder.php(1159): ReflectionClass->newInstanceArgs(Array)
#2 /home/vagrant/builtinx/vendor/symfony/dependency-injection/ContainerBuilder.php(623): Symfony\Component\DependencyInjection\ContainerBuilder->createService(Object(Symfony\Component\DependencyInjection\Definition), Array, 'console.chain_d...')
#3 /home/vagrant/builtinx/vendor/symfony/dependency-injection/ContainerBuilder.php(1264): Symfony\Component\DependencyInjection\ContainerBuilder->doGet('console.chain_d...', 1, Array)
#4 /home/vagrant/builtinx/vendor/symfony/dependency-injection/ContainerBuilder.php(1221): Symfony\Component\DependencyInjection\ContainerBuilder->doResolveServices(Object(Symfony\Component\DependencyInjection\Reference), Array)
#5 /home/vagrant/builtinx/vendor/symfony/dependency-injection/ContainerBuilder.php(1134): Symfony\Component\DependencyInjection\ContainerBuilder->doResolveServices(Array, Array)
#6 /home/vagrant/builtinx/vendor/symfony/dependency-injection/ContainerBuilder.php(623): Symfony\Component\DependencyInjection\ContainerBuilder->createService(Object(Symfony\Component\DependencyInjection\Definition), Array, 'console.chain')
#7 /home/vagrant/builtinx/vendor/symfony/dependency-injection/ContainerBuilder.php(588): Symfony\Component\DependencyInjection\ContainerBuilder->doGet('console.chain', 1)
#8 /home/vagrant/builtinx/vendor/drupal/console-core/src/Application.php(441): Symfony\Component\DependencyInjection\ContainerBuilder->get('console.chain')
#9 /home/vagrant/builtinx/vendor/drupal/console-core/src/Application.php(211): Drupal\Console\Core\Application->registerCommands()
#10 /home/vagrant/builtinx/vendor/drupal/console-core/src/Application.php(116): Drupal\Console\Core\Application->loadCommands()
#11 /home/vagrant/builtinx/vendor/drupal/console/src/Application.php(64): Drupal\Console\Core\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 /home/vagrant/builtinx/vendor/symfony/console/Application.php(148): Drupal\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#13 /home/vagrant/builtinx/vendor/drupal/console/bin/drupal.php(89): Symfony\Component\Console\Application->run()
#14 /home/vagrant/builtinx/vendor/drupal/console/bin/drupal(4): require('/home/vagrant/b...')
#15 {main}
ArgumentCountError: Too few arguments to function Drupal\Console\Core\Utils\ChainDiscovery::__construct(), 2 passed and exactly 4 expected in Drupal\Console\Core\Utils\ChainDiscovery->__construct() (line 83 of /home/vagrant/builtinx/vendor/drupal/console-core/src/Utils/ChainDiscovery.php).

With that I see where Drupal\Console\Dotenv\Generator\InitGenerator has no leading slash:
```
services:
  console.dotenv_init:
    class: \Drupal\Console\Dotenv\Command\InitCommand
    arguments: ['@app.root', '@?console.root', '@console.dotenv_init_generator']
    tags:
      - { name: drupal.command, bootstrap: uninstall }
  console.dotenv_debug:
    class: \Drupal\Console\Dotenv\Command\DebugCommand
    arguments: ['@app.root', '@?console.root']
    tags:
      - { name: drupal.command, bootstrap: uninstall }
  console.dotenv_init_generator:
    class: Drupal\Console\Dotenv\Generator\InitGenerator
    tags:
      - { name: drupal.generator, bootstrap: uninstall }
```

Versions:
Drupal version   : 8.5.3
Drupal Console version: 1.8.0
PHP version: 7.2.4

**Comment by vorapoap** [comment]
I have this problem too with Drupal8.7.4 DrupalConsole1.9.0

**Comment by podarok** [comment]
remove `composer.json` from drupal docroot folder

**Comment by enzolutions** [comment]
Solved, tested with version 1.9.3
